<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>uart_blck</class>
 <widget class="QWidget" name="uart_blck">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>439</width>
    <height>554</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout_2" stretch="6,1">
   <item>
    <layout class="QVBoxLayout" name="lay_show" stretch="1,1,1,1,1,1,1">
     <item>
      <widget class="QLabel" name="label_2">
       <property name="text">
        <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p align=&quot;center&quot;&gt;&lt;span style=&quot; font-size:22pt;&quot;&gt;串口配置&lt;/span&gt;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
       </property>
      </widget>
     </item>
     <item>
      <layout class="QHBoxLayout" name="lay_port_bit" stretch="1,3">
       <item>
        <widget class="QLabel" name="label_7">
         <property name="text">
          <string>端口号</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QComboBox" name="ui_port_bit"/>
       </item>
      </layout>
     </item>
     <item>
      <layout class="QHBoxLayout" name="lay_baud_bit" stretch="1,3">
       <item>
        <widget class="QLabel" name="label">
         <property name="text">
          <string>波特率</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QComboBox" name="ui_baud_bit"/>
       </item>
      </layout>
     </item>
     <item>
      <layout class="QHBoxLayout" name="lay_data_bit" stretch="1,3">
       <item>
        <widget class="QLabel" name="label_6">
         <property name="text">
          <string>数据位</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QComboBox" name="ui_data_bit"/>
       </item>
      </layout>
     </item>
     <item>
      <layout class="QHBoxLayout" name="lay_chck_bit" stretch="1,3">
       <item>
        <widget class="QLabel" name="label_4">
         <property name="text">
          <string>校验位</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QComboBox" name="ui_chck_bit"/>
       </item>
      </layout>
     </item>
     <item>
      <layout class="QHBoxLayout" name="lay_stop_bit" stretch="1,3">
       <item>
        <widget class="QLabel" name="label_5">
         <property name="text">
          <string>停止位</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QComboBox" name="ui_stop_bit"/>
       </item>
      </layout>
     </item>
     <item>
      <layout class="QHBoxLayout" name="lay_ctrl_bit" stretch="1,3">
       <item>
        <widget class="QLabel" name="label_3">
         <property name="text">
          <string>控制流</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QComboBox" name="ui_ctrl_bit"/>
       </item>
      </layout>
     </item>
    </layout>
   </item>
   <item>
    <layout class="QHBoxLayout" name="lay_button">
     <item>
      <widget class="QPushButton" name="ui_read_par">
       <property name="text">
        <string>读取</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="ui_setx_par">
       <property name="text">
        <string>配置</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="ui_exit_win">
       <property name="text">
        <string>退出</string>
       </property>
      </widget>
     </item>
    </layout>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
